src/shifts/shifts.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Shift } from './Shift';
+
+@Injectable()
+export class ShiftsService {
+    constructor(@InjectRepository(Shift) private shiftRepository: Repository<Shift>){}  
+
+    async createShift(item){
+        const {name , start_time , end_time} = item
+        let shiftsBeforeCreate = await this.shiftRepository.find()
+        await this.shiftRepository.save({name , start_time , end_time})
+        let shiftsAfterCreate = await this.shiftRepository.find()       
+        if (shiftsAfterCreate.length > shiftsBeforeCreate.length) {
+            return {message: "created"}
```

Comment:
I prefer not to use that naming start_time, please stick to the naming convention of the whole system

Reply:
I did it like this just to be able to work with the Angular ui